lib/opensrs/server.rb

```diff
@@ -91,6 +97,9 @@ def http
       end
       http.use_ssl = (server.scheme == "https")
       http.verify_mode = OpenSSL::SSL::VERIFY_NONE
```

Comment:
While we're here, can we remove this line and start verifying the server certificate? Right now, the connection is trivially vulnerable to man-in-the-middle attacks.

Reply:
Sounds very fishy indeed. I removed it.